### PR TITLE
add documentation

### PR DESCRIPTION
**Pull request by jeffrifwald**
@ryan-roemer @alexlande Let me know if any of this is unclear, as it is actually kind of a complicated process. Thanks!

Addresses #9.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,103 @@
...
+    },
+    "devDependencies": {
+      "eslint": "^1.0.0"
+    },
+    "scripts": {
+      "postpublish": "publishr postpublish",
+      "postversion": "publishr postversion"
+    },
+    "publishr": {
+      "dependencies": ["^babel"],
```
Comment (ryan-roemer) [inline README.md:77]: Why is it `^babel`? What does the caret mean?

**Comment by ryan-roemer** [comment]
@jmcriffey -- Great start here! Some additions I'd love
1. An example of actual shell commands after things are set up to actually publish.
2. A stronger callout that "you **must** use `npm version` workflow" with this.

File: README.md (hunk 1 above)
Comment (jeffrifwald) [inline README.md:77]: Good question. I didn't mention anywhere that the `dependencies` expects one or more regular expression strings. I've added some information that describes both the `dependencies` and `files` configurations.

**Comment by jeffrifwald** [comment]
@ryan-roemer This is ready for another review. Thanks!

**Comment by alexlande** [comment]
This is looking great. My only comment is that it would be nice to highlight the regular expression behavior a little bit more. I can imagine folks using this, passing in something like `["babel"]`, then being shocked when modules other than `babel` are missing.

**Comment by jeffrifwald** [comment]
Good catch @alexlande. I'll actually add in a whole section that explains the configurations. I'll use an example that shows you need `^babel$` to match only the `babel` package.

**Pull request merged**